shUdra-dharmaH/shUdra-kamalAkaraH_sAnuvAdaH/1_shUdradharmatattvam/02_adhikArasiddhiH.md

```diff
@@ -3,7 +3,7 @@ title = "०२ अधिकारसिद्धिः"
 
 +++
 ## मूलम्
-तत्र शूद्रत्वं उढसवर्णाजत्वम् । विप्रत्वादिवत् । यद्यपि सर्गाद्यकालीने विश्वामित्रादौ च नैतत्तथापि तत्रादृष्टविशेषारब्धशरीरत्वमेव ब्राह्मणवं । अन्यत्र तु पूर्वोक्तम् । एवं शूद्रेपीति दिक् । तत्र पद्भ्यां शूद्रो अजायत [ ऋ० सं० अ० ८ अ० ४ व० १९] । शूद्रो मनुष्याणामश्वः पशूनाम् [ तै० सं० कां० ७ अ० १ अ० १ ] । इति बहृचतैत्तिरीयादिश्रुतिभिः।। 
+तत्र शूद्रत्वं ऊढसवर्णाजत्वम् । विप्रत्वादिवत् । यद्यपि सर्गाद्यकालीने विश्वामित्रादौ च नैतत्तथापि तत्रादृष्टविशेषारब्धशरीरत्वमेव ब्राह्मणत्वं । अन्यत्र तु पूर्वोक्तम् । एवं शूद्रेपीति दिक् । तत्र पद्भ्यां शूद्रो अजायत [ ऋ० सं० अ० ८ अ० ४ व० १९] । शूद्रो मनुष्याणामश्वः पशूनाम् [ तै० सं० कां० ७ अ० १ अ० १ ] । इति बहुचतैत्तिरीयादिश्रुतिभिः।। 
```

Comment:
संस्कृतव्याकरणानुसारेण स्वरात् पूर्वं वाक्यान्ते वा ऽनुस्वारो न स्यात्। तद् भवत्या ऽनुष्ठेयम्। 

तथा, `बहुचतैत्तिरीयादिश्रुतिभिः` इत्यत्र सन्देहः। मूलाच् चित्रं दर्शयतु।

Reply:
तथा, बहुचतैत्तिरीयादिश्रुतिभिः इत्यत्र सन्देहः। मूलाच् चित्रं दर्शयतु। 
अत्र किञचित् स्पष्टता आवश्यकी।

Reply:
किम् अस्पष्टम् इति वदतु। मूलाच्चित्रं नाम किम् इति? screenshot of pdf corresponding to above lines.